Homework1.3

```diff
@@ -0,0 +1,10 @@
+// Call index outside of the bound. Handle this exception printing "Outside of bound message."
+
+class Main {
+    public static void main(String[]args){
+        int[]arrInt = {1,2,3,4,5}; {
+            System.out.println(arrInt[5]);
```

Comment:
You should handle your exception and print a message when you catch one. Please, wrap your code in try-catch block